src/CommunityToolkit.Maui.Core/Essentials/FolderPicker/FolderPickerImplementation.android.cs

```diff
@@ -18,8 +19,16 @@ public async Task<Folder> PickAsync(string initialPath, CancellationToken cancel
 		}
 
 		Folder? folder = null;
+		const string baseUrl = "content://com.android.externalstorage.documents/document/primary%3A";
+		if (Android.OS.Environment.ExternalStorageDirectory is not null)
+		{
+			initialPath = initialPath.Replace(Android.OS.Environment.ExternalStorageDirectory.ToString(), string.Empty);
```

Comment:
On my end, the Roslyn didn't like this call
![image](https://user-images.githubusercontent.com/20712372/219782299-0836ee5e-b0fe-4f44-ad74-463537994f7d.png)

And can we use the `AbsolutePath` property instead of `ToString` method?

Reply:
the problem was with thries parameter. string Comparison. Fixed